phpstan-baseline.neon

```
@@ -71965,3 +71965,75 @@ parameters:
 			message: "#^Call to an undefined method MauticPlugin\\\\MauticTagManagerBundle\\\\Security\\\\Permissions\\\\TagManagerPermissions\\:\\:expects\\(\\)\\.$#"
 			count: 1
 			path: plugins/MauticTagManagerBundle/Tests/Unit/Security/Permissions/TagManagerPermissionsTest.php
+
+		- '#\$id type mapping mismatch: database can contain string but property expects int.#'
+
+		-
+			message: "#^Property Mautic\\\\IntegrationsBundle\\\\Entity\\\\FieldChange\\:\\:\\$objectId type mapping mismatch: database can contain string but property expects int.$#"
+			count: 1
+			path: app/bundles/IntegrationsBundle/Entity/FieldChange.php
```

Can we resolve these issues rather than adding them on the top of the existing pile of ignored PHPSTAN errors? That's why we want to add the PHPSTAN-Doctrine extension in the first place, right? We should retype this property to be string. There should be no changes in this file. Or do you see it differently?

---

I researched this topic and found that doctrine treats `bigInt` as `string` because `bigInt` does not fit into php `int`, so we need a custom type to do the conversion as [discussed here](https://github.com/doctrine/dbal/issues/3690). 

I wanted to enforce `int` which is `4 bytes` in php until we upgrade `dbal` or until `phpstan doctrine` finds a solution for it.

---

We should switch to `string` as that's the right way how to handle MySQL's BIGINT as PHP's int can't handle the full range. We discussed this with @fedys some time ago. It would cause another rabbit hole of refactoring. But in this case you can simply accept that the `objectId` property will be int and getters and setters will retype it to int with a note that it has to be refactored later on.

---

I made the required changes for `bigint` on this [commit](https://github.com/mautic/mautic/pull/12262/commits/49f8a5b7f838b1bc7768f59e968939ba46593d73)

---

In this [commit](https://github.com/mautic/mautic/pull/12262/commits/ad66b80d6cd867e0bdd78bb9ce57b68a990305d7), we are back to what we have in `5.x` so no new ignored errors